content/posts/blog/2024-02-19-oci-runtime-spec-v1-2.md

```diff
@@ -0,0 +1,52 @@
+---
+title: OCI Runtime Spec v1.2
+author:
+  name: Open Container Initiative
+  tag: oci
+tags: blog
+date: 2024-02-19
+---
+
+We are delighted to annouce the release of the [OCI Runtime Spec](https://github.com/opencontainers/runtime-spec) v1.2.0.
+This release contains [about 10 pull requests](https://github.com/opencontainers/runtime-spec/milestone/19?closed=1) that were merged in the last three years.
```

Comment:
Looks more like 7 months, it might be easier to call out the previous release.

```suggestion
This release contains [about 10 pull requests](https://github.com/opencontainers/runtime-spec/milestone/19?closed=1) that were merged since the 1.1.0 release.
```